src/chaosbag/ChaosBagApi.ttslua

```diff
@@ -71,9 +71,9 @@ do
   ---@param drawAdditional boolean Controls whether additional tokens should be drawn
   ---@param tokenType? string Name of token (e.g. "Bless") to be drawn from the bag
   ---@param guidToBeResolved? string GUID of the sealed token to be resolved instead of drawing a token from the bag
-  ---@param returnedToken? tts__Object Token to be replaced with newly drawn token
+  ---@param takeParameters? table Position and rotation of the location where the new token should be drawn to, usually to replace a returned token
```

Comment:
Missing annotation for the new return value (object reference to the token?)

Reply:
Looks like that's unused now with the new redraw function taking its place.